develop/devguide/Connector/ConnectionsRedundantPolling.md

```diff
@@ -15,10 +15,26 @@ In the protocol, you need to define the 2 connections and specify that redundant
 The option "redundantPolling" in the communicationOptions attribute is required to activate the redundant polling mechanism.
 
 > [!NOTE]
-> Redundant polling is supported with protocols that define exactly two connections of the same type. The following is supported: two SNMP, two serial, two smart-serial or two HTTP connections.
+> Redundant polling is supported with protocols that define exactly two connections of the same type. The following is supported: two SNMP, two serial, two smart-serial, or two HTTP connections.
 >
 > Note that for smart-serial connections, in order to get a timeout (enabling the redundant polling connections to switch) the pair must contain a response. Alternatively, you can implement logic to set the communication state using the <xref:NT_CHANGE_COMMUNICATION_STATE> Notify call.
 
+## Switching implementation
```

Comment:
Are the cases described in this section the only cases when switching will ever be implemented, or could there be others?